src/test/java/steps/Google.java

```diff
@@ -0,0 +1,37 @@
+package steps;
```

Comment:
the Page is located in the steps package

Reply:
fixed